cmd/cluster/create.go

```diff
@@ -157,7 +165,18 @@ func CreateCluster(ctx context.Context, opts Options) error {
 	client := util.GetClientOrDie()
 
 	// Load or create infrastructure for the cluster
+	switch opts.Platform {
+	case "aws", string(v1alpha1.AWSPlatform):
+		return createInfraAWS(ctx, opts, pullSecret, sshKey, client, annotations)
+	case "none", "baremetal", string(v1alpha1.NonePlatform):
```

Comment:
please remove the term baremetal (https://github.com/openshift/hypershift/pull/452/files#r705149913) and let's use constants for platform names i.e None.